taskcluster/scripts/fetch-firefoxci-artifacts.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+
+import requests
+import taskcluster
+
+from fxci_config_taskgraph.util.integration import FIREFOXCI_ROOT_URL, STAGING_ROOT_URL
+
+if "TASKCLUSTER_PROXY_URL" in os.environ:
+    options = {"rootUrl": os.environ["TASKCLUSTER_PROXY_URL"]}
+else:
+    options = taskcluster.optionsFromEnvironment()
+    assert options["rootUrl"] == STAGING_ROOT_URL
+
+secrets = taskcluster.Secrets(options)
+
+result = secrets.get("project/releng/fxci-config/firefoxci-artifact-client")
+assert isinstance(result, dict)
+
+credentials = result["secret"]
+assert isinstance(credentials, dict)
+
+options = {"rootUrl": FIREFOXCI_ROOT_URL, "credentials": credentials}
```

Comment:
is this intended behavior? `options` is already set up above with `rootUrl`, and used to fetch credentials, and this one for the queue. Is there a use case where they would be using different `rootUrl`?

Reply:
AIUI, it first talks to staging to fetch the secret that contains the credentials for the Firefox CI cluster.

This might be a bit less confusing if there were two distinct `options` and/or a comment about the above.

Reply:
I'll add a comment.